README.md

```diff
@@ -1,2 +1,69 @@
 # data-collection
+
 Data and automated tools for data collection
+
+# Requirements
+
+- The device must have [`ffmpeg`](https://ffmpeg.org/download.html) installed
+- There must be a `praw.ini` file in the `reddit` folder
+- The `useful_functions` setup must be done (see the setup section below)
+
+# Example `praw.ini` file
+
+```{praw.ini}
+[bot1]
+client_id=TYPE_YOUR_CLIENT_ID_HERE
+client_secret=TYPE_YOUR_CLIENT_SECRET_HERE
+password=TYPE_YOUR_PASSWORD_HERE
+username=TYPE_YOUR_USERNAME_HERE
+user_agent=bot1
+```
+
+Details can be found in the
+[PRAW documentation](https://praw.readthedocs.io/en/stable/getting_started/configuration/prawini.html).
+
+# Setup
+
+## Setting up the Python environment
+
+If using Python venv, you can use the code below to set up the code:
+
+```{bash}
+python3 -m pip install -r requirements.txt
```

Comment:
you need to be in the venv first then once activated you can just do `pip install -r requirements.txt`

python3 -m venv venv makes the venv but when I deleted my venv and tried the line you gave I got a pip is unknown error

Reply:
I've [updated the instructions](https://github.com/myHaiven/data-collection/pull/4/commits/d46c06bdb765be986e747c8d497c066d62db7691) to tell the user to activate their environment first before doing the pip install.